src/main.rs

```diff
@@ -113,7 +114,10 @@ fn cmd() -> Command {
                        Command::new("list").about("List all tasks in a project")
                          .arg(config_arg())
                          .arg(project_arg())
-                         .arg(flag_arg("scheduled", 's',  "Only list tasks that are scheduled for today and have a time")),
+                         .arg(flag_arg("scheduled", 's',  "Only list tasks that are scheduled for today and have a time"))
+                         .arg(flag_arg("done-today", 't',  "Only list tasks that were done today"))
+                         .arg(flag_arg("due-today", 'x',  "Only list tasks that are due today"))
+                         .arg(flag_arg("done-yesterday", 'd',  "Only list tasks that were done yesterday")),
```

Comment:
Running out of flags sucks! I don't have any strong opinions about the character choices and don't mind if we want to break some later.

Reply:
For now I just left it as it is, I just don't see the options.
Will rework in future, maybe will change some clap code